app/models/ability.rb

```diff
@@ -0,0 +1,22 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+
+    user ||= User.new
+
+    can :read, [Product]
+    can [:read,:create], Review
```

Comment:
Not logged in user can leave comments. But the difference between his comments from the logged-in user`s comments is that after approve by the admin, his (not logged in user`s) comment does not have the Verified mark.